doc/source/multiprocessing.rst

```diff
@@ -7,6 +7,9 @@ multiprocessing.Pool API (Experimental)
   so it may be changed at any time without warning. If you encounter any
   bugs/shortcomings/incompatibilities, please file an `issue on GitHub`_.
   Contributions are always welcome!
+  
+.. warning::
+  The 'context' argument passed to pool \_\_init\_\_ function is not supported using ray.
```

Comment:
```suggestion
  The ``context`` argument to the ``Pool`` constructor is not supported using ray.
```

I think having a top-level warning for this is excessive, maybe put it in the section below where we say "The full API is currently supported..." Also, please specify that you can still pass the argument but it is ignored.